python/ucxx/ucxx/_lib/arr.pyx

```diff
@@ -295,3 +297,18 @@ cdef inline Py_ssize_t _nbytes(Py_ssize_t itemsize,
     for i in range(ndim):
         nbytes *= shape_mv[i]
     return nbytes
+
+
+cpdef Array asarray(obj):
+    """Coerce other objects to ``Array``. No-op for existing ``Array``s.
+
+    Args:
+        obj: Object exposing the Python buffer protocol or ``__cuda_array_interface__``
+
+    Returns:
+        Array: An instance of the ``Array`` class
```

Comment:
```suggestion
    Parameters
    ----------
    obj: object
        Object exposing the Python buffer protocol or ``__cuda_array_interface__``.

    Returns
    -------
    array: Array
        An instance of the ``Array`` class.
```

Reply:
Let's also attempt to keep the docstrings formatting used by the rest of the project.